static/app/views/performance/traceDetails/transactionDetail.tsx

```diff
@@ -81,7 +81,7 @@ class TransactionDetail extends Component<Props> {
           {tn(
             '%s error event occurred in this transaction.',
             '%s error events occurred in this transaction.',
-            errors.length
+            errors.length + performance_issues.length
```

Comment:
We should change the text here, since it wouldn't be accurate to refer to performance issues as error events

Reply:
Went with `issue` to match the trace navigator:

<img width="375" alt="image" src="https://user-images.githubusercontent.com/4205004/229605577-a479a3d6-10ab-4acf-bf64-358b1afe991a.png">

Reply:
Nice! 😄